script.js

```diff
@@ -0,0 +1,154 @@
+class Piece {
+  Shapes = {
+    "stair-top-right": [
+      [1, 1],
+      [0, 1],
+    ],
+    "stair-top-left": [
+      [1, 1],
+      [1, 0],
+    ],
+    "stair-bottom-left": [
+      [1, 0],
+      [1, 1],
+    ],
+    "stair-bottom-right": [
+      [0, 1],
+      [1, 1],
+    ],
+    "upper-slab": [
+      [1, 1],
+      [0, 0],
+    ],
+    "lower-slab": [
+      [0, 0],
+      [1, 1],
+    ],
+    empty: [],
+  };
+
+  constructor(shape) {
+    this.shape = shape;
+  }
+
+  toArray() {
+    return this.Shapes[this.shape];
+  }
+}
+
+class Glyph {
+  Alphabet = {
```

Comment:
This is a change that's been made twice now and I'm intruiged by the motivation behind it? My hard-typed soul makes me think this is an accident waiting to happen! Interested to know your thoughts